.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
         "out": true // set this to false to include "out" folder in search results
     },
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-    "typescript.tsc.autoDetect": "off"
+    "typescript.tsc.autoDetect": "off",
+    "python.pythonPath": "C:\\Users\\Vandy\\AppData\\Local\\Programs\\Python\\Python38-32\\python.exe"
```

Comment:
Looks like you accidentally committed this! :O

Reply:
i don't know what ur talking about 😅

jk, GOOD CATCH THANK YOU